crates/subspace-node/src/core_domain/cli.rs

```diff
@@ -95,6 +96,25 @@ impl CoreDomainCli {
 
         cli
     }
+
+    /// Creates domain configuration from Core domain cli.
+    pub fn create_domain_configuration(
+        &self,
+        tokio_handle: tokio::runtime::Handle,
+    ) -> sc_cli::Result<DomainConfiguration> {
+        // if is dev, use the known key ring to start relayer
+        let maybe_relayer_id = if self.shared_params().is_dev() {
+            self.run.get_keyring().map(|kr| kr.to_account_id())
+        } else {
+            self.relayer_id.clone()
+        };
```

Comment:
We should use `self.relayer_id` and only fall back to default in dev mode if it is `None`. Otherwise you can't override default behavior.

Reply:
yeah, this was mostly taking easiet route instead of another check assuming its highly unlikely we wont override that 🤣 . But i do see a someone is going to be frustrated when they override and wont work